pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -166,6 +202,36 @@ func (r *ReconcileWindowsMachineConfig) getCloudProvider(instance *wmcapi.Window
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.Info("reconciling", "namespace", request.Namespace, "name", request.Name)
+	wmcoObject := false
+	// Fetch the Machine instance
+	Machine := &mapiv1.Machine{}
+	provisionedPhase := "Provisioned"
+	if err := r.client.Get(context.TODO(), request.NamespacedName, Machine); err != nil {
+		wmcoObject = true
+		// TODO: Uncomment this when we switch to Machine-api completely
```

Comment:
Please add `WINC-429` to do this comment, so that it will be easy for whoever picks up `WINC-429`. I am adding a few TODOs referencing this also.

Reply:
Ok